components/service/glean/README.md

```diff
@@ -129,6 +129,75 @@ adb shell am start -n org.mozilla.samples.glean/mozilla.components.service.glean
   --es sendPing metrics
 ```
 
+## Unit testing glean metrics
```

Comment:
How about...
- we instead add a short section on testing below [adding new metrics](https://github.com/mozilla-mobile/android-components/blob/master/components/service/glean/README.md#adding-new-metrics)
- we move this detailed information to e.g. `docs/metrics/testing-metrics.md`

Reply:
As is this is rather long for an overview README.

Reply:
Good idea!  I just pushed this change, adding a short summary in the README.md with a link to `docs/metrics/testing-metrics.md` which now contains the full description and examples.